src/api/apiFetch.js

```diff
@@ -42,24 +42,25 @@ export const apiCall = async (
     networkActivityStart(isSilent);
     const response = await apiFetch(auth, route, params);
 
-    if (!response.ok) {
-      console.log('Bad response for:', { auth, route, params, response }); // eslint-disable-line
+    const json = await response.json();
```

Comment:
That is definitely a change we do not want to make.
Look at the history of this file to see the motivation.
Short: if the response is HTML this will throw a misleading error.